### DISCUSSION TITLE
Gappy mesh for shoe lasts

### DISCUSSION THREAD

**Discussion by trilbytim** [alicevision/Meshroom#2082] [Show and tell]
My mum needs special orthopaedic shoes made and some time ago had lasts made of her feet. They are made of wood and leather, I'd like to 3D print copies of these which can be sent off without risk of losing the originals.
![image](https://github.com/alicevision/Meshroom/assets/42208042/db77b8e1-0fde-40df-98c9-69359ee6c0c7)
I've taken about 160 images from all around this using my Samsung S9 phone camera. The StructureFromMotion stage looks OK with no large gaps (I went back 4 times adding images with 'augment reconstruction' to fill in when there were gaps)
![image](https://github.com/alicevision/Meshroom/assets/42208042/7ff99451-0642-4d89-b7c1-3a84e1b6e24b)
However when I get to trying to create the mesh it leaves a lot of holes:
![image](https://github.com/alicevision/Meshroom/assets/42208042/23bfd8f9-d3f0-4c9e-88d1-875a9723036a)
Also instead of fitting a smooth surface across the outside, the mesher ends up creating cavities inside that look like stalactites when you look inside the mesh:
![image](https://github.com/alicevision/Meshroom/assets/42208042/bef210f8-c88c-46bc-b65d-64f99445ff46)
I'd be happy with the overall mesh density if it would fit a better surface over them. What can I try to get a better mesh for printing?

**Comment by Zvyozdo4ka** [comment]
use as minimal aperture as possible, put on shoe markers, like stickers, then retake photographs, and try to build model again
